app/commands/decidim/translation_addons/create_report.rb

```diff
@@ -30,18 +33,28 @@ def create_report
           @current_user,
           visibility: "public-only"
         ) do
-          report = Decidim::TranslationAddons::Report.new(
-            decidim_user_id: current_user.id,
-            decidim_resource_type: resource_instance.class.name,
-            decidim_resource_id: resource_instance.id,
-            reason:,
-            field_name: field,
-            locale:,
-            fix_suggestion:
-          )
+          report = Decidim::TranslationAddons::Report.where(decidim_resource_id: resource_instance.id, decidim_resource_type: resource_instance.class.name, field_name: field,
+                                                            locale:).first
+          if report.blank?
+            report = Decidim::TranslationAddons::Report.new(
+              decidim_resource_type: resource_instance.class.name,
+              decidim_resource_id: resource_instance.id,
+              field_name: field,
+              locale:
+            )
+            report.save!
+          end
```

Comment:
`first_or_create` or `first_or_initialize` just does that.

```suggestion
          report = Decidim::TranslationAddons::Report.where(decidim_resource_id: resource_instance.id, decidim_resource_type: resource_instance.class.name, field_name: field,
                                                            locale:).first_or_create
```